frontend/conf/routes

```diff
@@ -59,6 +59,8 @@ GET         /masterclasses/:tag/:subTag                controllers.WhatsOn.maste
 
 # Event Details
 GET         /event/:id                                 controllers.Event.details(id)
```

Comment:
I left the old /event/ URL alone, but changed the logic in the method to redirect if it determines the event is definitely a Live Event or Masterclass - I don't think there is likely to be any non-specific RichEvents, but if there is this is the path to render them.